### ISSUE TITLE
unbalance performance between forward and backward generator

### ISSUE THREAD

**Issue by fedral** [junyanz/pytorch-CycleGAN-and-pix2pix#1225]
Hi,   
     I trained cyclegan from scratch on dataset facade and used same parameter setting & train code.
After 220+ epoch,  the forward label-to-image generator works as expected, but the backward image-to-label generator performed poorly compared with forward mapping.  [ backward loss is almost 10 times compared to forward loss]

Could provide some suggestions to address this unbalance.

![image](https://user-images.githubusercontent.com/13436512/104669581-47796580-5715-11eb-88cd-107a19e4084f.png)

**Comment by JunlinHan** [comment]
Solution 1: Increase the weights for idt loss and G loss for mapping (image2label).
Solution2 : change your training epochs to default setting. ( 200 epochs in total)
